modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java

```diff
@@ -7,32 +7,37 @@
 
 import com.liferay.portal.kernel.exception.PortalException;
 
+import java.util.List;
+
 /**
  * @author Marco Leo
  */
 public class ObjectRelationshipEdgeException extends PortalException {
 
-	public ObjectRelationshipEdgeException() {
```

Comment:
Hey @larissacribeiro, I think the pattern we follow is to keep the constructors generated by buildService. Can you check other examples and if I'm correct, fix this?

Reply:
We discussed this in person with other devs and aligned that it was okay to remove the unused ones, it has already been done other in other files. thanks!